tests/Security/TwoFactor/Trusted/TrustedDeviceTokenTest.php

```diff
@@ -18,19 +21,31 @@ class TrustedDeviceTokenTest extends TestCase
 
     protected function setUp(): void
     {
-        $jwtToken = $this->createMock(Token::class);
-        $jwtToken
-            ->expects($this->any())
-            ->method('getClaim')
-            ->willReturnMap([
-                [JwtTokenEncoder::CLAIM_USERNAME, false, 'username'],
-                [JwtTokenEncoder::CLAIM_FIREWALL, false, 'firewallName'],
-                [JwtTokenEncoder::CLAIM_VERSION, false, 1],
-            ]);
-        $jwtToken
-            ->expects($this->any())
-            ->method('__toString')
-            ->willReturn('serializedToken');
+        if (class_exists(Plain::class)) {
+            // lcobucci/jwt v4
+            $claims = new DataSet([
+                JwtTokenEncoder::CLAIM_USERNAME => 'username',
+                JwtTokenEncoder::CLAIM_FIREWALL => 'firewallName',
+                JwtTokenEncoder::CLAIM_VERSION => 1,
+            ], 'encodedClaims');
+
+            $jwtToken = new Plain(new DataSet([], 'encodedHeaders'), $claims, Signature::fromEmptyData());
+        } else {
+            // lcobucci/jwt v3
+            $jwtToken = $this->createMock(Token::class);
+            $jwtToken
+                ->expects($this->any())
+                ->method('getClaim')
+                ->willReturnMap([
+                    [JwtTokenEncoder::CLAIM_USERNAME, false, 'username'],
+                    [JwtTokenEncoder::CLAIM_FIREWALL, false, 'firewallName'],
+                    [JwtTokenEncoder::CLAIM_VERSION, false, 1],
+                ]);
+            $jwtToken
+                ->expects($this->any())
+                ->method('__toString')
+                ->willReturn('encodedHeaders.encodedClaims.');
+        }
```

Comment:
Also looks like a leftover from the previous implementation. Could be removed, right?